system/Model.php

```diff
@@ -348,6 +348,42 @@ public function find($id = null)
 		return $row['data'];
 	}
 
+	/**
+	 * Fetches the column of database from $this->table with a primary key
+	 * matching $id.
+	 *
+	 * @param mixed|array|null $id One primary key or an array of primary keys
+	 * @param mixed|array|null $column_name Column name or null for primaryKey
+	 *
+	 * @return array|null array of table's column values
+	 */
+	public function findColumn($id = null, string $column_name = null) : array
+	{
+
+		if(is_null($column_name))
+			$column_name = $this->primaryKey;
+
+		if(!is_string($column_name))
+			throw DataException::forFindColumnIsNotAString();
+
+		$this->select($column_name);
+		$tmp = $this->asArray()->find($id);
+
+		if(count($tmp))
+		{
+			$data = [];
+
+			foreach ($tmp as $item)
+			{
+				$data[] = $item[$column_name];
+			}
+
+			return $data;
+		}
+
+		return null;
```

Comment:
No need to specify `null`. Just `return` does it

Reply:
true..

Reply:
But... I like to specify return value which is ?array  for now.... and when I am using return instead of return null I get:
```
A function with return type must return a value (did you mean "return null;" instead of "return;"?) 
```

Reply:
That's interesting, and good to know. In the past they've always been treated the same, but I've never tried specifying it with an optional return type either. Cool - let it ride.